diablo/jobs/kaltura_job.py

```diff
@@ -485,18 +491,22 @@ def _mark_success(schedule_updates, field_names, kaltura_schedule_id=None):
             su.mark_success()
 
 
-def _mark_error(schedule_updates, e, message, field_names, kaltura_schedule_id=None):
+def _mark_error(schedule_updates, e, message, field_names):
```

Comment:
Removed the logic in this method requiring a match on `kaltura_schedule_id`, as a schedule ID mismatch might itself be the source of the error.